projects/portal/src/app/views/dialogs/incentive/withdraw-incentive-all-rewards-form-dialog/withdraw-incentive-all-rewards-form-dialog.component.css

```diff
@@ -0,0 +1,3 @@
+.collapse {
+  visibility: visible !important;
+}
```

Comment:
It is added to use DaisyUI's collapse.
Since there is an "important: true," setting in tailwind.config.js, it is hidden by the same name item of collapse in tailwind's CSS, so it is added.